### DISCUSSION TITLE
Make all functions global by default and use `noGlobal` to turn off?

### DISCUSSION THREAD

**Discussion by slmjkdbtl** [replit/kaboom#152] [Ideas]
Want to make kaboom most appealing to beginners. Wondering if it's better to make global by default (like p5.js?), and have option to use context handle instead

```js
kaboom()

// just start, clean
add()
action()
keyPress()
```

with `noGlobal`
```js
const k = kaboom({
    noGlobal: true,
});

k.add();
k.action();
k.keyPress();
```

**Comment by lajbel** [comment]
Well, for programmers to experiment it would be something strange, but for people new to the world it would be nice not to have to configure that and simply start using the functions

**Comment by slmjkdbtl** [comment]
I made this change in beta and feels pretty good, it's not breaking since the handle is always returned, only `noGlobal` if you're afraid of polluting namespace.
